deblur/test/data/testmerge.fastq.trim.derep.no_artifacts.msa.deblur.no_chimeras

```diff
@@ -0,0 +1,20 @@
+>810205;size=1
```

Comment:
This file is called FASTQ but it is still a FASTA? I think it will make more sense to do the test with an actual FASTQ, just to make sure that it works as expected.